src/Honeybee_ExportEPC.py

```diff
@@ -127,18 +140,48 @@ def main(HBZones):
     # return surface data
     return srfData
     # done!
-
-
+#print srfData
 srfTypeDict = {0: "Wall", 1 : "Roof", 2: "Floor", 5 : "Glazing"}
 directionDict = {0: "N", 1 : "NW", 2: "W", 3: "SW", 4 : "S", 5 : "SE", 6 : "E", 7 : "NE"}
 
+
```

Comment:
@pchopson sorry for being unresponsive for the last few days. We are trying to push a new release out which should happen at some point tonight/tomorrow. I will get back to you asap. Thanks.